src/write.c

```diff
@@ -947,6 +960,24 @@ size_t avifEncoderGetGainMapSizeBytes(avifEncoder * encoder)
 {
     return encoder->data->gainMapSizeBytes;
 }
+
+// Sets altImageMetadata's metadata values to represent the "alternate" image after applying the gain map to the base image.
```

Comment:
Maybe use "as if" instead of "after" because at first I thought the function would "apply the gain map to the base image".